src/platforms/common/enriching-events/scopes.mdx

```diff
@@ -93,13 +93,17 @@ You can also apply this configuration when unsetting a user at logout:
 To learn what useful information can be associated with scopes see
 [the context documentation](../context/).
 
-<PlatformSection>
-
 ## Local Scopes
 
 We also support pushing and configuring a scope within a single call. This is typically
-called `with-scope` or `push-scope` depending on the SDK, and is very helpful if
-you only want to send data for one specific event. In the following example we use
+called `with-scope`, `push-scope` or implemented as a function parameter on the capture methods, depending on the SDK. It's very helpful if
+you only want to send data for one specific event.
+
+<PlatformSection supported={["go", "java", "javascript", "native", "php", "python", "ruby", "rust"]}>
```

Comment:
How did we arrive at this list of supported platforms? Is it all that have a file in `src/includes/enriching-events/scopes/with-scope`?

Reply:
Native just has `_This is not supported by the Native SDK._` shouldn't that simply be removed from the list?

Reply:
Yes, went through the platforms in the with-scope folder.

Removing native sounds good to me. Will have a look if that has any implications.